nginx/content.md

```diff
@@ -42,9 +42,7 @@ You can mount your configuration file, or build a new image with it.
 If you wish to adapt the default configuration, use something like the following to get it from a running nginx container:
 
 ```console
-$ docker run --name tmp-nginx-container -d %%IMAGE%%
-$ docker cp tmp-nginx-container:/etc/nginx/nginx.conf /host/path/nginx.conf
-$ docker rm -f tmp-nginx-container
+$ docker run --rm --entrypoint=cat nginx /etc/nginx/nginx.conf > /host/path/nginx.conf
```

Comment:
Doh sorry, didn't notice sooner -- this should be `%%IMAGE%%` so the templating uses the correct version on the per-arch pages too:

```suggestion
$ docker run --rm --entrypoint=cat %%IMAGE%% /etc/nginx/nginx.conf > /host/path/nginx.conf
```

Reply:
(I'm happy to make this change for you if you'd prefer)

Reply:
Good catch!